src/OctoshiftCLI.Tests/GithubClientTests.cs

```diff
@@ -156,7 +224,7 @@ public async Task GetAsync_Logs_The_Response_Status_Code_And_Content()
         public async Task GetAsync_Throws_HttpRequestException_On_Non_Success_Response()
         {
             // Arrange
-            using var httpResponse = new HttpResponseMessage(HttpStatusCode.InternalServerError);
+            var httpResponse = () => new HttpResponseMessage(HttpStatusCode.InternalServerError);
```

Comment:
same. I would just suppress the warning here, because not disposing the response message in test doesn't matter at all.